src/task-loop.ts

```diff
@@ -25,7 +26,7 @@ export class TaskLoop {
     start = () => {
         if (!this.started) {
             this.started = true;
-            this.handle = setTimeout(this.runTask, this.pause);
+            this.handle = setTimeout(this.runTask, 0);
```

Comment:
nice...

Reply:
yeah, so make the tests easier, no reason to wait 30 seconds on the first iteration in staging